index.js

```diff
@@ -9,65 +9,96 @@ const newMember = require("./data/new-member");
  * @param {Object} obj -
  * @return {number} The number of keys in the object
  */
-function numberOfKeys(obj) {}
+function numberOfKeys(obj) {
+  return _.keys(obj).length;
+}
 
 /**
  * Remove the falsy values in a numbers array and return the sum
  * @param {numbers[]} array - An array of numbers that can also contain some falsy values
  * @return {number} The sum of the numbers in an array
  */
-function sumNumbers(array) {}
+function sumNumbers(array) {
```

Comment:
The use of _.sum() is appropriate, but _.remove() is not the correct method to filter out falsy values. Consider using _.compact() instead.